### PR TITLE
chore: add oracle to docker-live

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

**Comment by kleyow** [comment]
@lewisdaly I know nothing about helm :| I just inferred what needed to be added so double check this for me

**Comment by lewisdaly** [comment]
Thanks @kleyow this is a good start - it doesn't look too far off to me. 

Leave it to me to deploy and test - I'm doing some other PISP helm stuff this week so it fits in nicely

File: docker-live/charts-base/deployment_setup.yaml [hunk 1]
```diff
@@ -140,6 +140,70 @@ spec:
...
+    metadata:
+      labels:
+        app: als-consent-oracle-mysql
+    spec:
+      containers:
+      - image: mysql:5.6
+        name: als-consent-oracle-mysql
+        env:
+        - name: MYSQL_ROOT_PASSWORD
+          value: password
```
Comment (eoln) [inline docker-live/charts-base/deployment_setup.yaml:187]: maybe we should keep secrets in env not in chart... ?

**Review by eoln — DISMISSED** [review]
lgtm -> but we need to think how we are storing secrets in charts

**Comment by lewisdaly** [comment]
> lgtm -> but we need to think how we are storing secrets in charts

Agreed - this can be tackled when we move things over from docker-local to mojaloop/helm

**Comment by lewisdaly** [comment]
Thanks @kleyow - I made a few tweaks, and it's mostly working.

I've created https://github.com/mojaloop/project/issues/2015 as a follow up - we need the FSPIOP-API to support the `CONSENTS` type as well/

**Review by lewisdaly — APPROVED** [review]

**Pull request merged**